samples/nfc/tnep_tag/src/main.c

```diff
@@ -33,12 +29,8 @@ static const uint8_t svc_two_uri[] = "svc:e";
 static uint8_t tag_buffer[NDEF_TNEP_MSG_SIZE];
 static uint8_t tag_buffer2[NDEF_TNEP_MSG_SIZE];
 
-NFC_NDEF_MSG_DEF(ndef_msg, 16);
+static const uint8_t en_code[] = {'e', 'n'};
 
-NFC_NDEF_TEXT_RECORD_DESC_DEF(nfc_text, UTF_8, en_code, sizeof(en_code),
-			      msg, sizeof(msg));
-NFC_NDEF_TEXT_RECORD_DESC_DEF(svc_one_rec, UTF_8, en_code, sizeof(en_code),
-			      svc_one_msg, sizeof(svc_one_msg));
 NFC_NDEF_TEXT_RECORD_DESC_DEF(svc_two_rec, UTF_8, en_code, sizeof(en_code),
```

Comment:
Can we also make this record descriptor automatic?

Reply:
It can be I moved it to local scope